sdk/attestation/azure-security-attestation/README.md

```diff
@@ -127,15 +127,15 @@ clients to add, remove or enumerate the policy management certificates.
 
 ## Examples
 
-- [Instantiate a synchronous attestation client](#create-a-synchronous-attestation-client)
-- [Retrieve token validation certificates](#retrieve-token-certificates)
+- [Create a synchronous attestation client](#create-a-synchronous-attestation-client)
+- [Retrieve Token Certificates](#retrieve-token-certificates)
```

Comment:
This API retrieves the certificates needed for attestation token validation. IMHO, calling them Token Certificates simplifies this too much, they should still be called token validation certificates.